src/Swagger.ObjectModel/Primitive.cs

```diff
@@ -17,6 +17,7 @@ public class Primitive
             // Numbers
             { typeof(float), new Primitive("number", "float") },
             { typeof(double), new Primitive("number", "double") },
+            { typeof(decimal), new Primitive("number", "decimal") },
```

Comment:
You could do a SwaggerTypeMapping from `decimal?` to `decimal`. 
`SwaggerTypeMapping.AddTypeMapping(typeof(decimal?), typeof(decimal));`.
The class was created to handle similar cases, plus a few others. Though we should probably add some logic to handle nullables properly. 

But I think adding decimal to the primitives is fine, because it is a primitive. And as long as the type is 'number', swagger will handle it.